gruntfileConfig.json

```diff
@@ -4,6 +4,7 @@
   "src": "./src",
   "assets": "./src/assets",
   "theme": "./node_modules/patternpack-example-theme",
+  "logo": "/theme-assets/images/pp-logo.svg",
```

Comment:
How would the user get the logo into the `/theme-assets/images` folder without creating a new theme?  I thought the idea here, was for a user to pick an existing theme, and just add their logo to it.  Am I missing something?